### DISCUSSION TITLE
Why My Pull Shark Badge didn't contain full history for all contribution ?

### DISCUSSION THREAD

**Discussion by SherifMega** [community/community#57734] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

For Pull shark badge
i have contribution in react-navigation package and that's displayed correctly.

but new contribution to shopify flash-list not appear in pull shark history.

any ideas to display it ?

**Comment by danielalyoshin** [comment, marked as answer]
As far as I know, the badge will only show the PRs that led you to acquire each tier of that badge. This means that the badge will only show your 2nd, 16th, 128th, and 1024th PR, each corresponding to the acquisition of a tier.

Hope this helps!
